src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetDateCommand.cs

```diff
@@ -369,9 +370,7 @@ private string UFormatDateString(DateTime dateTime)
                             break;
 
                         case 'c':
-                            sb.Append("{0:ddd} {0:MMM} ");
-                            sb.Append(StringUtil.Format("{0,2} ", dateTime.Day));
-                            sb.Append("{0:HH}:{0:mm}:{0:ss} {0:yyyy}");
+                            sb.Append("{0:ddd} {0:dd} {0:MMM} {0:HH}:{0:mm}:{0:ss} {0:yyyy}");
```

Comment:
Shouldn't this be

```c#
"{0:ddd} {0:dd} {0:MMM} {0:yyyy} {0:HH}:{0:mm}:{0:ss} {0:tt}"
```

Reply:
If we add {0:tt} should we use {0:hh}?

Reply:
I thought that the positioning of the month and day `({0:dd} and {0:MMM})` were locale dependent. It is for POSIX anyway (http://pubs.opengroup.org/onlinepubs/009604599/utilities/date.html). This routine might need to be a bit more clever

Reply:
The problem is that we need to emulate UNIX behavior based on Unix locales to get the same that datetime returns but we can use only CoreFX locales.
We can get standard format strings:
```powershell
[CultureInfo]::new("en-US").DateTimeFormat
```
There I don't see that we can use.

I test on WSL
```
LC_TIME="en_US.UTF-8" date +%c
LC_TIME="ru_RU.UTF-8" date +%c
```
 and get the same format for all locales. Could you test on other Unix?
So I suggest use ""{0:ddd} {0:dd} {0:MMM} {0:yyyy} {0:HH}:{0:mm}:{0:ss}" as a compromise.

Reply:
@iSazonov looks like you didn't make this change (the year is still at the end)?

Reply:
Done.